src/material/list/selection-list.spec.ts

```diff
@@ -337,35 +337,35 @@ describe('MatSelectionList without forms', () => {
     it('should restore focus if active option is destroyed', () => {
       const manager = selectionList.componentInstance._keyManager;
 
-      spyOn(listOptions[2].componentInstance, 'focus').and.callThrough();
-      listOptions[3].componentInstance._handleFocus();
+      spyOn(listOptions[3].componentInstance, 'focus').and.callThrough();
```

Comment:
Some of these unrelated tests had to be tweaked, because I added an extra option to one of the fixtures.